pandas/tests/test_format.py

```diff
@@ -2952,6 +2952,16 @@ def test_to_csv_date_format(self):
         self.assertEqual(df_day.to_csv(), expected_default_day)
         self.assertEqual(df_day.to_csv(date_format='%Y-%m-%d'), expected_default_day)
 
+        # GH 7791
```

Comment:
put in a short explanation. also try with some of the other options (``nan_rep`` comes to mind) as well.